app/plugin/FeatureInfoWindow.js

```diff
@@ -0,0 +1,143 @@
+Ext.define('CpsiMapview.plugin.FeatureInfoWindow', {
+    extend: 'Ext.plugin.Abstract',
+    alias: 'plugin.cmv_feature_info_window',
+    pluginId: 'cmv_feature_info_window',
+
+    requires: [
+        'Ext.Component',
+        'CpsiMapview.view.window.MinimizableWindow',
+        'BasiGX.view.grid.FeaturePropertyGrid',
+        'BasiGX.util.Map'
+    ],
+
+    init: function () {
+        var me = this;
+
+        var mapComp = BasiGX.util.Map.getMapComponent();
```

Comment:
It is actually used as plugin for `CpsiMapview.view.main.Map` which then contains the map component. But it is also possible to use it as a plugin for the `gx_map`. This might make more sense.